src/wcl/xoshiro256.h

```diff
@@ -20,7 +20,7 @@
 #include <cstdint>
 #include <tuple>
 
-#pragma once
```

Comment:
it's not shown in the diff, but `#pragma once` was listed twice

Reply:
oh my bad lol

Reply:
Xoshiro256 should be renamed `xoshiro_256` to fit the wcl naming convention.

Reply:
Also you should change `step` to be `operator()` instead so that it can be used with C++ <random> distributions. This is basically like mersene_twister but with a tiny fraction of the memory foot print, vastly improved speed, and improved statistical properties. It would be nice to use it with `std::uniform_int_distriution` and `std::uniform_real_distribution` in the future I think.